vtex/omnishipping
# Fixes wrongfully sending invoiceData when user is in delivery

#### What is the purpose of this pull request?
- Fixes wrongfully sending invoiceData when user is in delivery

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/21583/erro-403-por-envio-de-invoicedata-mesmo-se-usu%C3%A1rio-n%C3%A3o-est%C3%A1-interagindo-com-essa-se%C3%A7%C3%A3o)

#### How should this be manually tested?
1. Add [items to cart](https://fernando--vtexgame1invoice.myvtex.com/checkout/cart/add/?sku=297&qty=1&seller=1&sc=1)
2. Identify as shipping6.680867846933269@mailinator.com
3. Try to proceed to payment and should do it successfully

#### Screenshots or example usage
n/a

#### Types of changes
